R/layer_population_scaling.R

```diff
@@ -0,0 +1,133 @@
+#' Revert population scaled prediction
+#'
+#' @param frosting a `frosting` postprocessor
+#' @param ... the column(s) in the `epi_df` to scale back.
+#' @param df a data frame that contains the population data used for scaling.
+#' @param by A character vector of variables to join by. Column names should be
+#' in lower case.
+#' To join by different variables on the `epi_df` and `df`, use a named vector.
+#' For example, by = c("geo_value" = "states") will match `epi_df$geo_value`
+#' to `df$states`. To join by multiple variables, use a vector with length > 1.
+#' For example, by = c("geo_value" = "states", "county" = "county") will match
+#' `epi_df$geo_value` to `df$states` and `epi_df$county` to `df$county`.
```

Comment:
We should say what `NULL` the default does.

Reply:
added description and also unit tests for the default case.